src/styles/blocks/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  width: $card-width;
+  border: 1px solid $c-light-gray;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  color: $c-black;
+  font-size: $font-basic-size;
+  line-height: 14px;
+
+  &__image {
+    display: flex;
+    margin: auto;
+    margin-bottom: 40px;
+  }
+
+  &__tittle {
+    margin-bottom: 4px;
+    line-height: 18px;
+    font-size: $font-basic-size;
+    font-weight: $font-weight_medium;
+  }
+
+  &__review,
+  &__code,
+  &__price {
+    font-weight: 400;
+  }
+
+  &__review,
+  &__code {
+    font-size: 10px;
+  }
+
+  &__code,
+  &__price {
+    color: $c-gray;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+  }
```

Comment:
Better keep properties with the corresponding style.
Your code is hard to understand and maintain.

Reply:
Ok, done